cql3/statements/alter_keyspace_statement.cc

```diff
@@ -144,15 +144,14 @@ void add_prefixed_key(const sstring& prefix, const std::map<sstring, sstring>& i
 };
 
 std::map<sstring, sstring> get_current_options_flattened(const shared_ptr<cql3::statements::ks_prop_defs>& ks,
-                                                         bool include_tablet_options,
                                                          const gms::feature_service& feat) {
     std::map<sstring, sstring> all_options;
 
     add_prefixed_key(ks->KW_REPLICATION, ks->get_replication_options(), all_options);
     add_prefixed_key(ks->KW_STORAGE, ks->get_storage_options().to_map(), all_options);
     // if no tablet options are specified in ATLER KS statement,
     // we want to preserve the old ones and hence cannot overwrite them with defaults
```

Comment:
I don't understand what this comment is saying, or if it's still true after this patch.

Reply:
The comment is still valid but one needs to look at the following lines as well: https://github.com/scylladb/scylladb/blob/027cd6daff5cd8b6fb7c2b5e517367fae1bcf126/cql3/statements/alter_keyspace_statement.cc#L200-L202

The "old options" are created from the current keyspace metadata, whereas the "current options" from properties of the ALTER KEYSPACE statement. We do not want to override tablet options accidently to default values when `AND tablets {...}` is not provided with the statement..

Reply:
ping @nyh